test/constants/helpers/schema.json

```diff
@@ -3105,7 +3105,7 @@
           "description": "Mutations for a person who uses our app",
           "fields": [
             {
-              "name": "searchResultId",
```

Comment:
My guess is because `npm test` generates it, and @Kumjami has done `git add .` or something similar. I don't actually know what this schema is used for, but since it's generated then it *probably* should be added to .gitignore.